app/controllers/api/auth/stories_controller.rb

```diff
@@ -45,11 +46,37 @@ def filtered(resources)
   end
 
   def resources_base
-    # If there is a network_id specified, use that network
+    # If there is a network_id specified, use that network.
+    # The authz is performed in check_user_network().
     @stories ||= if params[:network_id]
       super.published
     else
       authorization.token_auth_stories
     end
   end
+
+  def search
+    # same logic as resources_base above
+    @stories ||= if params[:network_id]
+      Story.text_search(search_query, search_params)
+    else
+      Story.text_search(search_query, search_params, authorization)
+    end
+    index
+  end
+
+  def search_params
+    sparams = super
+    sparams[:fq] ||= {}
+    [:network_id, :series_id, :account_id, :app_version].each do |p|
```

Comment:
Just to note ... it's different that these are all specified via the `?q` param instead of coming from the route or the `?v4` filter.  Not bad, just different.

Also a bit awkward that the JSON representers render camelCase, but the search params (and a lot of our current query filters/sorts/etc) are under_scored.  But that's a bigger problem for another day.

Reply:
I think they are coming from the Rails `params` which are populated from the URL routing. So not from the `q` param. Unless I am misunderstanding?

Reply:
Ah, I see.  So someday when we alter the routes to include sub-searches (like `/api/v1/series/123/stories`), those will start working.  But with this PR, the only way to use them is via the `?q=series_id:123` param.

Reply:
I see these routes defined (and tested).
```
                                   search_api_stories GET     /api/:api_version/stories/search(.:format)                                                              api/stories#search {:api_version=>"v1"}
             search_api_authorization_account_stories GET     /api/:api_version/authorization/accounts/:account_id/stories/search(.:format)                           api/auth/stories#search {:api_version=>"v1"}
              search_api_authorization_series_stories GET     /api/:api_version/authorization/series/:series_id/stories/search(.:format)                              api/auth/stories#search {:api_version=>"v1"}
                     search_api_authorization_stories GET     /api/:api_version/authorization/stories/search(.:format)                                                api/auth/stories#search {:api_version=>"v1"}
             search_api_authorization_network_stories GET     /api/:api_version/authorization/networks/:network_id/stories/search(.:format)                           api/auth/stories#search {:api_version=>"v1"}
```

Reply:
Whoops!  I must have just missed those.